### DISCUSSION TITLE
Would the integration sync with a Mi account?

### DISCUSSION THREAD

**Discussion by Nickduino** [dckiller51/bodymiscale#240] [Q&A] [answered]
I don't have Bluetooth on my HA server and I use Zepp life with my Mi scale. Could I still use the integration to get data from my Mi account and display them in HA? Or would another integration be more suitable?

**Comment by dckiller51** [comment, marked as answer]
> Je n'ai pas de Bluetooth sur mon serveur HA et j'utilise Zepp life avec ma balance Mi. Puis-je quand même utiliser l'intégration pour récupérer les données de mon compte Mi et les afficher dans HA ? Ou une autre intégration serait-elle plus adaptée ?

Hi, sorry there is no link to the application. This component emulates the results of the zepp application. Weight and impedance are required.
